exchanges/kraken/kraken_wrapper.go

```diff
@@ -485,53 +485,52 @@ func (k *Kraken) UpdateTradablePairs(ctx context.Context, forceUpdate bool) erro
 	return k.EnsureOnePairEnabled()
 }
 
+func (k *Kraken) newPairFromSymbol(symbol string, item asset.Item) (currency.Pair, error) {
+	cp, err := k.MatchSymbolWithAvailablePairs(symbol, item, item == asset.Futures)
+	if err != nil {
+		if errors.Is(err, currency.ErrPairNotFound) {
```

Comment:
Might be better to invert this check and return the error, just so if there is any funny things going on with that function above instead of potentially returning an empty pair and nil. 

```
if !errors.Is(err, currency.ErrPairNotFound) {
   return err
}
// Lookup here.
```

Reply:
Much nicer!